src/System.Diagnostics.TraceSource/tests/TraceTestHelper.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Text;
+
+namespace System.Diagnostics.TraceSourceTests
+{
+    static class TraceTestHelper
+    {
+        /// <summary>
+        /// Resets the static state of the trace objects before a unit test.
+        /// </summary>
+        public static void ResetState()
+        {
+            Trace.Listeners.Clear();
+            Trace.Listeners.Add(new DefaultTraceListener());
+            Trace.AutoFlush = false;
+            Trace.IndentLevel = 0;
+            Trace.IndentSize = 4;
+            Trace.UseGlobalLock = true;
+            // Trace holds on to instances through weak refs
+            // this is intented to clean those up.
+            GC.Collect();
+            Trace.Refresh();
+            TraceSwitch.RefreshAll();
+        }
+
+        public static String NewLine
+        {
+            get
+            {
+                return new StringBuilder().AppendLine().ToString();
+                // NOTE: Want to use this line:                
+                //return System.Environment.NewLine;
+                // However, I get this compile error, that I have absoltely no idea how to resolve.
```

Comment:
You can add System.Runtime.Extensions to the packages.config, essentially adding a reference to System.Runtime.Extensions.dll, in order to get access to System.Environment.